pwndbg/commands/config.py

```diff
@@ -42,24 +45,37 @@ def config():
     for v in sorted(values):
         print_row(v.optname, repr(v.value), repr(v.default), v.docstring, longest_optname, longest_value)
 
+    print(light_yellow('You can set config variable with `set <config-var> <value>`'))
+    print(light_yellow('You can generate configuration file using `configfile` '
+                       '- then put it in your .gdbinit after initializing pwndbg'))
+
+
 @pwndbg.commands.Command
 def configfile():
     """Generates a configuration file for the current Pwndbg options"""
     configfile_print_scope('config')
 
+
 @pwndbg.commands.Command
 def themefile():
     """Generates a configuration file for the current Pwndbg theme options"""
     configfile_print_scope('theme')
 
+
 def configfile_print_scope(scope):
-    values = [v for k, v in pwndbg.config.__dict__.items()
-              if isinstance(v, pwndbg.config.Parameter) and v.scope == scope]
-    longest_optname = max(map(len, [v.optname for v in values]))
-    longest_value = max(map(len, [extend_value_with_default(repr(v.value), repr(v.default)) for v in values]))
+    filter_cond = lambda v: \
```

Comment:
Separately, I agree with the style guide in PEP8 that suggests breaking *before* the operator.

```
if foo \
and bar:
```

versus:

```
if foo and \
bar:
```

(per https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)